modules/serverless-ossm-installing-and-configuring-openshift-service-mesh.adoc

```diff
@@ -0,0 +1,148 @@
+:_content-type: PROCEDURE
+[id="serverless-ossm-installing-and-configuring-openshift-service-mesh_{context}"]
+= Installing and configuring {SMProductShortName}
+
+To integrate {ServerlessProductShortName} with {SMProductShortName}, you need to install {SMProductShortName} with a specific configuration.
+
+.Procedure
+
+. Create a `ServiceMeshControlPlane` resource in the `istio-system` namespace with the following configuration:
++
+[IMPORTANT]
+====
+If you have an existing `ServiceMeshControlPlane`, make sure that you have the same configuration applied.
+====
++
+[source,yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+  namespace: istio-system
+spec:
+  profiles:
+  - default
+  security:
+    dataPlane:
+      mtls: true <1>
+  techPreview:
+    meshConfig:
+      defaultConfig:
+        terminationDrainDuration: 35s <2>
+  gateways:
+    ingress:
+      service:
+        metadata:
+          labels:
+            knative: ingressgateway <3>
+  proxy:
+    networking:
+      trafficControl:
+        inbound:
+          excludedPorts: <4>
+          - 8444 # metrics
+          - 8022 # serving: wait-for-drain k8s pre-stop hook
+----
+<1> Enforce strict mTLS in the mesh. Only calls using a valid client certificate are allowed.
+<2> {ServerlessProductShortName} has a graceful termination for Knative Services of 30 seconds. `istio-proxy` needs to have a longer termination duration to make sure no requests are dropped.
+<3> Define a specific selector for the ingress gateway to target only the Knative gateway.
+<4> These ports are called by Kubernetes and cluster monitoring, which are not part of the mesh and cannot be called using mTLS. Therefore, these ports are excluded from the mesh.
+
+. Add the namespaces that you would like to integrate with {SMProductShortName} to the `ServiceMeshMemberRoll` object as members:
++
+.Example `servicemesh-member-roll.yaml` configuration file
+[source,yaml]
+----
+apiVersion: maistra.io/v1
+kind: ServiceMeshMemberRoll
+metadata:
+  name: default
+  namespace: istio-system
+spec:
+  members: <1>
+    - knative-serving
+    - knative-eventing
+    - your-OpenShift-projects
+----
+<1> A list of namespaces to be integrated with {SMProductShortName}.
++
+[IMPORTANT]
+====
+This list of namespaces must include the `knative-serving` and `knative-eventing` namespaces.
+====
+
+. Apply the `ServiceMeshMemberRoll` resource:
++
+[source,terminal]
+----
+$ oc apply -f servicemesh-member-roll.yaml
+----
+
+. Create the necessary gateways so that {SMProductShortName} can accept traffic. The following example uses `knative-local-gateway` object using ISTIO_MUTUAL (mTLS):
```

Comment:
Markup "ISTIO_MUTUAL". Also seems like a few words might be missing. Suggestion, if still technically accurate:
```suggestion
. Create the necessary gateways so that {SMProductShortName} can accept traffic. The following example uses the `knative-local-gateway` object with the `ISTIO_MUTUAL` mode (mTLS):
```